apps/prairielearn/src/middlewares/authzCourseOrInstance.ts

```diff
@@ -161,7 +161,7 @@ export async function authzCourseOrInstance(req: Request, res: Response) {
   res.locals.institution = authzData.institution;
 
   // The side nav is expanded by default.
-  res.locals.side_nav_expanded = req.session?.side_nav_expanded ?? true;
+  res.locals.side_nav_expanded = req.session.side_nav_expanded ?? true;
```

Comment:
The optional chaining here was deliberate; the session middleware intentionally doesn't run for API routes. This is causing the failing test.